Kitodo-DataManagement/src/main/java/org/kitodo/data/elasticsearch/search/Searcher.java

```diff
@@ -65,16 +67,12 @@ public Long countDocuments() throws DataException {
      */
     public Long countDocuments(String query) throws DataException {
         SearchRestClient restClient = initiateRestClient();
-        JSONParser parser = new JSONParser();
 
         String response = restClient.countDocuments(overrideNullQuery(query));
         if (!response.equals("")) {
-            try {
-                JSONObject result = (JSONObject) parser.parse(response);
-                return (Long) result.get("count");
-            } catch (ParseException e) {
-                throw new DataException(e);
-            }
+            JsonReader jsonReader = Json.createReader(new StringReader(response));
```

Comment:
`JsonReader` is a closeable class but the created object is never closed. Maybe you can use a try-with-resources statement?